### ISSUE TITLE
bug(outdated): -webkit-text-stroke and unicode-bidi properties

### ISSUE THREAD

**Issue by Kristinita** [rossPatton/stylint#436]
### 1. Summary

Stylint show, that `-webkit-text-stroke` and `unicode-bidi` properties as invalid.

### 3. Can I use

+ [**`-webkit-text-stroke`**](https://caniuse.com/#search=-webkit-text-stroke)
+ [**`unicode-bidi` and `direction`**](https://caniuse.com/#search=unicode-bidi)

### 4. Environment

+ Stylint 1.5.9

### 5. Configuration

`SashaValidStylus.styl`:

```stylus
.Sasha404
    -webkit-text-stroke .1rem red

noscript
    >span
        unicode-bidi bidi-override
        direction rtl

```

### 6. Steps to reproduce

    stylint SashaVaildStylus.styl

### 7. Expected behavior

No warnings and errors in console.

### 8. Actual behavior

```shell
D:\Kristinita>stylint SashaVaildStylus.styl
SashaVaildStylus.styl
2:1 valid warning property is not valid

SashaVaildStylus.styl
4:0 valid warning property is not valid

Stylint: 0 Errors.
Stylint: 2 Warnings.
```

Thanks.
